### DISCUSSION TITLE
Does Saleor support multiple store merchants?

### DISCUSSION THREAD

**Discussion by lonnietc** [saleor/saleor#12837] [General]
Hi,

I am researching various eCommerce software now and am wondering if Saleor supports multiple independent merchants as well?

Actually, I am seeking the case that:

1. Customers can search across all merchant stores
2. eCommerce platform will support multiple merchants.

Any information would be appreciated.
Thanks in advance

**Comment by timuric** [comment]
@lonnietc yes you can use multiple vendors and split them by channels, you can provide channel permissions to avoid admins of different channels observing each other's data

**Comment by albertojuniorhc** [comment]
This may be a silly question, but I'm kind of struggling with how to handle different channels with multiple locales.

I would like to have a multi-store / multi-brand environment

Example:
BRAND_A -> should be available in US, CA, FR
BRAND_B -> should be available in US, CA, FR

Knowing that each channel only supports one currency, do I need to create 6 different channels? Or maybe there is another way to handle this?

**Reply by timuric** [reply]
@albertojuniorhc correct, you would need dedicated channel for each currency. So if you have 2 brands in 3 different markets, you would need 6 channels.
